src/github.com/stellar/go-horizon/resource_account.go

```diff
@@ -13,11 +16,17 @@ import (
 // AccountResource is the summary of an account
 type AccountResource struct {
 	halgo.Links
-	ID          string            `json:"id"`
-	PagingToken string            `json:"paging_token"`
-	Address     string            `json:"address"`
-	Sequence    int64             `json:"sequence"`
-	Balances    []BalanceResource `json:"balances"`
+	ID            string             `json:"id"`
+	PagingToken   string             `json:"paging_token"`
+	Address       string             `json:"address"`
+	Sequence      int64              `json:"sequence"`
+	Numsubentries int32              `json:"numsubentries"`
+	InflationDest null.String        `json:"inflation_destination"`
+	HomeDomain    null.String        `json:"home_domain"`
+	Thresholds    map[string]byte    `json:"thresholds"`
```

Comment:
IMO, we should only be using maps for json when we cannot build use a go type to store the data.  The only cases I can think of for this are when the keys are variable, such as the "details" attributes for operations and effects.  In this case, I think it would be better to have:

```go
type ThresholdsResource struct {
    LowThreshold byte `json:"low_threshold"`
    MedThreshold byte `json:"med_threshold"`
    HighThreshold byte `json:"high_threshold"`
}
```